theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer.tsx

```diff
@@ -119,8 +111,9 @@ export class TraceViewerWidget extends ReactWidget {
 
     dispose(): void {
         super.dispose();
+        signalManager().off(Signals.OUTPUT_ADDED, (payload: OutputAddedSignalPayload) => this.onOutputAdded(payload));
         signalManager().off(Signals.EXPERIMENT_SELECTED, (experiment: Experiment) => this.onExperimentSelected(experiment));
-        signalManager().off(Signals.TRACEVIEWER_CLOSED, (UUID: string) => this.onCloseExperiment(UUID));
+        signalManager().off(Signals.CLOSE_TRACEVIEWERTAB, (UUID: string) => this.onCloseExperiment(UUID));
```

Comment:
The listener does not get removed because it is not the same instance being passed that was added earlier. (3x)